mongomock/aggregate.py

```diff
@@ -488,6 +489,35 @@ def _handle_type_convertion_operator(self, operator, values):
                 'You need to import the pymongo library to support decimal128 type.'
             )
 
+        # Document: https://docs.mongodb.com/manual/reference/operator/aggregation/toDecimal/
+        if operator == '$toDecimal':
+            if not decimal_support:
+                raise NotImplementedError(
+                    'You need to import the pymongo library to support decimal128 type.'
+                )
+            parsed = self.parse(values)
+            decimal_value = None
+            if isinstance(parsed, bool):
+                parsed = '0' if parsed is True else '1'
```

Comment:
I think this is reversed. You'll catch this if you add a test in test__mongomock

Reply:
You're right, my mistake